crates/stages/src/trie/mod.rs

```diff
@@ -54,17 +54,33 @@ where
         Ok(<Self as cita_trie::DB>::get(self, key)?.is_some())
     }
 
-    fn insert(&self, key: Vec<u8>, value: Vec<u8>) -> Result<(), Self::Error> {
-        // Caching and bulk inserting shouldn't be needed, as the data is ordered
-        self.tx.put::<tables::AccountsTrie>(H256::from_slice(key.as_slice()), value)?;
+    fn insert(&self, _key: Vec<u8>, _value: Vec<u8>) -> Result<(), Self::Error> {
+        unreachable!("Use batch instead.");
+    }
```

Comment:
since it's a trait impl, both removing and adding the `deprecated` doesn't work. the latter will throw `help: remove the unnecessary deprecation attribute` error.

i think in the future we're revamping this struct to take a cursor instead, and so, this can actually be used.